sdks/python/setup.py

```diff
@@ -155,7 +155,7 @@ def cythonize(*args, **kwargs):
 # Exclude 1.5.0 and 1.5.1 because of
 # https://github.com/pandas-dev/pandas/issues/45725
 dataframe_dependency = [
-    'pandas>=1.4.3,!=1.5.0,!=1.5.1,<2.3;python_version>="3.8"',
+    'pandas>=1.4.3,!=1.5.0,!=1.5.1,<2.3;python_version>="3.9"',
```

Comment:
we can omit python_version>="3.9" since now it includes all the versions we support

Reply:
conditions like this make sense when we install different dependency versions depending on Python version, for example, pandas 1.x for Python 3.7, and Pandas 2.x for all other versions.

Reply:
fixed